src/main/java/graphql/GraphQLContext.java

```diff
@@ -42,6 +42,12 @@ public class GraphQLContext {
 
     private final ConcurrentMap<Object, Object> map;
 
+    /**
+     * The key that should be associated with a boolean value which indicates whether @defer and @stream behaviour is enabled for this execution.
+     */
+    @ExperimentalApi
+    public static final String ENABLE_INCREMENTAL_SUPPORT  = "ENABLE_INCREMENTAL_SUPPORT";
+
```

Comment:
I think we should love this anotehr class - yes the mechanism is this - but the static should be elsewhere

One thought is this

```
public @interface ExperimentalApi {
    String ENABLE_INCREMENTAL_SUPPORT  = "ENABLE_INCREMENTAL_SUPPORT";
}

```
which then allows

```
GraphQLContext.newContext().put(ExperimentalApi.ENABLE_INCREMENTAL_SUPPORT,true);
```

Reply:
That makes sense 👍  I'm making the change